arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -446,40 +482,43 @@ class ffmpeg_video_input::priv
       }
 
       // grab the metadata from this packet if from the metadata stream
-      auto md_iter = this->metadata.find(this->f_packet->stream_index);
-      if (md_iter != this->metadata.end() )
+      auto md_iter = this->metadata.find( this->f_packet->stream_index );
+      if( md_iter != this->metadata.end() )
       {
         md_iter->second.emplace(
-          this->f_packet->pts,
-          std::deque<uint8_t>(this->f_packet->data,
-                              this->f_packet->data + this->f_packet->size));
+            this->f_packet->pts,
+            std::deque< uint8_t >( this->f_packet->data,
+                                   this->f_packet->data +
+                                   this->f_packet->size ) );
```

Comment:
So... this:
```suggestion
            { this->f_packet->data, 
              this->f_packet->data + this->f_packet->size } );
```
...is shorter, but here there's a better argument to be made for keeping the explicit type name, so this is less a suggestion and more a "what do you think about...?".